From 87172ae62c79dcd5b384d1a916dc06f81ebc23e2 Mon Sep 17 00:00:00 2001
From: Imagination Technologies <powervr@imgtec.com>
Date: Mon, 25 Sep 2017 15:58:49 +0100
Subject: [PATCH 020/168] egl: eglBindAPI workaround for dEQP bug

dEQP relies on eglBindAPI to only return true if the API can
successfully be used to create contexts, which the spec does not
require.
Until dEQP is fixed, just disable GL on non-X11 platforms.
---
 src/egl/main/eglcurrent.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/egl/main/eglcurrent.h b/src/egl/main/eglcurrent.h
index d813a46d9ab..c03798eaeac 100644
--- a/src/egl/main/eglcurrent.h
+++ b/src/egl/main/eglcurrent.h
@@ -72,7 +72,7 @@ struct _egl_thread_info
 static inline EGLBoolean
 _eglIsApiValid(EGLenum api)
 {
-#ifdef ANDROID
+#ifndef HAVE_X11_PLATFORM
    /* OpenGL is not a valid/supported API on Android */
    return api == EGL_OPENGL_ES_API;
 #else
-- 
2.17.1

